WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Preview/ProductDetailPreviewViewModel.swift

```diff
@@ -32,3 +84,91 @@ final class ProductDetailPreviewViewModel: ObservableObject {
         // TODO
     }
 }
+
+private extension ProductDetailPreviewViewModel {
+    func observeGeneratedProduct() {
+        generatedProductSubscription = $generatedProduct
+            .compactMap { $0 }
+            .sink { [weak self] product in
+                guard let self else { return }
+                self.updateProductDetails(with: product)
+            }
+    }
+
+    func updateProductDetails(with product: Product) {
+        productName = product.name
+        productDescription = product.fullDescription ?? product.shortDescription ?? self.productDescription
```

Comment:
❓ Is there a reason to assign `self.productDescription` as a fallback? 

I am asking because I don't think we will save this `self.productDescription` value when we save the product, which might be confusing. 

Please correct me if I am missing something.

Reply:
Removed in 1b65617b1a0d0026751c42308d7b32f47442b827.